.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+## Title
```

Comment:
Good job!

Reply:
like this, here you can see the comment directly in vscode

Reply:
Ok :) - I wish there was any soulution for radibuttons in Markdown, but for now you have to remove the answer you dont need. Or we could just have inserted <!-- Answer yes or no -->...

Reply:
True, radio buttons would be awesome in markdown! But I think the solution you came up with is a great way around it